### ISSUE TITLE
Let's remove vulnerability issues, by upgrading the request module?

### ISSUE THREAD

**Issue by KhaledMohamedP** [sidorares/yandex-translate#16]
There seems to be issue with the current version of `request` in `yandex-translate` with high severity. Please check the following list for more information: 

https://nodesecurity.io/check/yandex-translate

```
yandex-translate@2.1.1 > request@2.68.0 > hawk@3.1.3 > hoek@2.16.3
yandex-translate@2.1.1 > request@2.68.0 > tough-cookie@2.2.2
yandex-translate@2.1.1 > request@2.68.0 > tough-cookie@2.2.2
```

There is a current PR #11 which should address this issue, but it needs to be reviewed and tested.

**Comment by sidorares** [comment]
hi @baitun  , not very actively but I'll try time to go through PRs and bump outdated dependencies

**Comment by sp90** [comment]
@sidorares that would be amazing 👍

**Comment by sp90** [comment]
@sidorares  would you please run `npm publish` so we can use the updated version? 👍

**Comment by coltongit** [comment]
Considering that the action specified in this issue report has been done for a few months now, it should be OK to close this issue. @KhaledMohamedP / @sidorares, either of you can go ahead and do so.

**Comment by sidorares** [comment]
@KhaledMohamedP @coltongit published yandex-translate@2.1.3 with bumped request dep
